pillar/heroku/mit_open.sls

```diff
@@ -0,0 +1,234 @@
+{% set rds_endpoint = salt.boto_rds.get_endpoint('rds-postgresql-mitopen') %}
+
+{% set etl_micromasters_host = salt.sdb.get('sdb://consul/open-{}-etl-micromasters-host'.format(environment)) %}
+{% set etl_xpro_host = salt.sdb.get('sdb://consul/open-{}-etl-xpro-host'.format(environment)) %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  app_name: {{ env_data.app_name }}
+  api_key: __vault__::secret-operations/global/heroku/odl-devops-api-key>data>value
+  config_vars:
+    AKISMET_API_KEY: __vault__::secret-{{ business_unit }}/global/akismet>data>api_key
+    AKISMET_BLOG_URL: https://discussions-rc.odl.mit.edu
+    ALGOLIA_API_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>api_key
+    ALGOLIA_APP_ID: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>app_id
+    ALLOWED_HOSTS: '["*"]'
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws/creds/mitopen>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws/creds/mitopen>data>secret_key
+    AWS_STORAGE_BUCKET_NAME: 'ol-mitopen-{{ env_data.env_name }}'
+    CELERY_WORKER_MAX_MEMORY_PER_CHILD: {{ env_data.CELERY_WORKER_MAX_MEMORY_PER_CHILD }}
+    CKEDITOR_ENVIRONMENT_ID:  __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/ckeditor>data>environment_id
+    CKEDITOR_SECRET_KEY:  __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/ckeditor>data>secret_key
+    CKEDITOR_UPLOAD_URL:  __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/ckeditor>data>upload_url
+    CSAIL_BASE_URL: https://cap.csail.mit.edu/
+    DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/mitopen
+    DEBUG: {{ env_data.DEBUG }}
+    DUPLICATE_COURSES_URL: https://raw.githubusercontent.com/mitodl/open-resource-blacklists/master/duplicate_courses.yml
+    EDX_API_ACCESS_TOKEN_URL: https://api.edx.org/oauth2/v1/access_token
+    EDX_API_CLIENT_ID: __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/edx-api-client>data>id
+    EDX_API_CLIENT_SECRET: __vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/edx-api-client>data>secret
+    EDX_API_URL: https://api.edx.org/catalog/v1/catalogs/10/courses
+    EMBEDLY_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/embedly_key>data>value
+    ENABLE_INFINITE_CORRIDOR: {{ env_data.ENABLE_INFINITE_CORRIDOR }}
+    FEATURE_ANONYMOUS_ACCESS: True
+    FEATURE_ARTICLE_UI: True
+    FEATURE_COMMENT_NOTIFICATIONS: True
+    FEATURE_COURSE_FILE_SEARCH: True
+    FEATURE_COURSE_UI: {{ env_data.FEATURE_COURSE_UI }}
+    FEATURE_SPAM_EXEMPTIONS: True
+    FEATURE_INDEX_UPDATES: True
+    FEATURE_MOIRA: True
+    FEATURE_PODCAST_APIS: True
+    FEATURE_PODCAST_SEARCH: True
+    FEATURE_PROFILE_UI: True
+    FEATURE_SAML_AUTH: True
+    FEATURE_SEARCH_UI: True
+    FEATURE_USE_NEW_BRANDING: True
+    FEATURE_WIDGETS_UI: True
```

Comment:
These can be removed.

Reply:
all of them ?

Reply:
I removed all of FEATURE_* but they can easily be restored if it turns out they are needed.